.travis.yml

```diff
@@ -104,3 +104,8 @@ notifications:
             - secure: "A93qvTOlwlMK5WoEvZQ5jQ8Z4Hd0JpeO53WYt8iIJ3s/L6AubkfiN7gwhThRtPnPx7DVMenoKRMlcRg76/ICvXEViVnGgXFjsypF0CzVcIay9pPdjpZjZHP735yLfX512RtxYEdEGwi5r25Z2CEFaydhhxNwfuMxGBtLUjusix4="
         use_notice: true
         skip_join: true
+
+branches:
```

Comment:
Please move this section to the top of the file, and add a comment explaining what that regex is trying to capture.